src/components/pages/Thanks.tsx

```diff
@@ -28,6 +30,36 @@ export const Thanks: React.FC = () => {
     <>
       <br />
       <Container>{JSON.stringify(data)}</Container>
+      <Card style={{ width: '18rem' }}>
+    <Card.Body>
+      <Media>
+      <img
+        width={64}
+        height={64}
+        src="https://cdn.discordapp.com/avatars/480160173361725451/43a19472064aed12d00311b7de8fa67b.webp"
+        />
+        <img
+        width={54}
```

Comment:
All the img tags should have an alt attribute unique to the subject of the photo 
maybe you could probably use the username of the person who the image displays.

Reply:
You should also make the image round with aphrodite Stylesheet you 
could use the border radius property to do this. https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius